### DISCUSSION TITLE
Asterisk Patch auf Fritz!Box 7490

### DISCUSSION THREAD

**Discussion by snake225** [Freetz-NG/freetz-ng#931] [Off-topic]
Hallo liebe Forengemeinde,

ich interessiere mich für den Asterisk Patch in Verbindung mit Freetz Image auf einer Fritz!Box 7490.Ich möchte eine Telefonanlage mit Cisco IP Telefonen die mit dem SCCP- Protokoll funktionieren aufbauen. 
Ist mein Vorhaben realisierbar?

Mir wurde empfohlen mich an FDA zu wenden. 
Laut Aussage soll Asterisk nicht eingebaut werden können,wegen iksemel Fehler.

könnte mir jemand hier mehr Informationen hierzu bereitstellen?

Vielen Dank im Voraus für Eure Hilfe.

Mit freundlichen Grüßen

Ronny

**Comment by fda77** [comment]
Die Addon-Scheisse aus dem DEB wird hier nicht supported, frag dort nach wo dein Image her ist:  https://freetz-ng.github.io/freetz-ng/ADDONS.html
